sample_vehicle_description/config/simulator_model.param.yaml

```
@@ -16,3 +16,4 @@
     steer_time_constant: 0.27 # time constant of the 1st-order steering dynamics
     x_stddev: 0.0001 # x standard deviation for dummy covariance in map coordinate
     y_stddev: 0.0001 # y standard deviation for dummy covariance in map coordinate
+    enable_road_slope_simulation: true # if true, slopes in the lanelet map are used to apply an extra acceleration to the ego vehicle
```

```suggestion
    enable_road_slope_simulation: false # if true, slopes in the lanelet map are used to apply an extra acceleration to the ego vehicle
```
@maxime-clem the default behavior is setting the slope compensation to false, I think it is good to have this param on the yaml, but it should be false by default